kobo/client/commands/cmd_create_worker.py

```diff
@@ -0,0 +1,34 @@
+import sys
+from xmlrpc.client import Fault
+import pprint
+
+from kobo.client import ClientCommand
+
+
+class Create_Worker(ClientCommand):
+    """create a worker"""
+    enabled = True
+    admin = True
+
+
+    def options(self):
+        self.parser.usage = "%%prog %s [--create] worker_name [worker_name]" % self.normalized_name
+
+    def run(self, *args, **kwargs):
+        create = kwargs.pop("create", False)
```

Comment:
What does the `create` flag do?

It doesn't seem to do anything (and it seems redundant since the command name already starts with 'create').

Can it be removed?

Reply:
I accidentally left it while experimenting. I have removed it now.